packages/jest-util/src/isInteractive.ts

```diff
@@ -2,4 +2,4 @@
 
 import isCI from 'is-ci';
 
-export default process.stdout.isTTY && process.env.TERM !== 'dumb' && !isCI;
+export default !!process.stdout.isTTY && process.env.TERM !== 'dumb' && !isCI;
```

Comment:
nice

Reply:
type of `undefined | true` according to TS - probably correct 🙂